app/models/user.rb

```diff
@@ -117,16 +117,15 @@ def can_access_id_card?
   end
 
   def self.from_merged_attrs(existing_user, new_user)
```

Comment:
mentioned this in slack but the method name could change now that it's expiring the old user

Reply:
actually could probably all be done in sessions_controller since the concept of existing_user vs new_user becomes somewhat irrelevant.